application.py

```diff
@@ -1,3 +1,10 @@
+import os
+
+if os.getenv("NOTIFY_ENVIRONMENT") in {"development", "local", "preview"}:
+    import newrelic.agent
+
+    newrelic.agent.initialize("newrelic.ini")
```

Comment:
i think we need to pass the environment in here to appropriately enable/disable the tracing?

```suggestion
    newrelic.agent.initialize("newrelic.ini", os.getenv("NOTIFY_ENVIRONMENT"))
```

Reply:
I was choosing to use the `NEW_RELIC_x` env vars for now, eg `NEW_RELIC_ENVIRONMENT`.

Reply:
1) No, it doesn't crash if you omit the LICENSE_KEY or most settings. Config errors we can hard-fail on, we will (which I had done through a config var but will now hardcode).
2) It will initialise with some default values, which is why I want to have the default monitor_mode as off so that the risk of anything happening if we're misconfigured is lower.
3) Have ripped out all env vars except the license key.